front/src/styles/variables.css

```diff
@@ -1,9 +1,12 @@
 :root {
     --main-bg-color:  #6663FF;
+    --ff-secondary-variant-color: #8F96A3;
```

Comment:
we don't have this color in palette

Reply:
secondary variant is #FB9400 https://www.figma.com/file/QGYATYyXIUHW5a0j7dx65s/%D0%9F%D1%80%D0%BE%D0%B5%D0%BA%D1%82-%D0%A4%D0%B8%D0%BB%D1%8C%D0%BC%D1%8B?node-id=1870%3A8066